qa-core/src/tests/internal-api/tables/tables.spec.ts

```diff
@@ -86,4 +86,61 @@ describe("Internal API - Application creation, update, publish and delete", () =
     //Table was deleted
     await config.tables.getAll(2)
   })
+
+  it("Search and pagination", async () => {
+    // create the app
+    const appName = generator.word()
+    const app = await createAppFromTemplate()
+    config.applications.api.appId = app.appId
+
+    // Get current tables: expect 2 in this template
+    await config.tables.getAll(2)
+
+    // Add new table
+    const [createdTableResponse, createdTableData] = await config.tables.save(
+      generateTable()
+    )
+
+    //Table was added
+    await config.tables.getAll(3)
+
+    //Get information about the table
+    await config.tables.getTableById(<string>createdTableData._id)
+
+    //Add Column to table
+    const newColumn = generateNewColumnForTable(createdTableData)
+    const [addColumnResponse, addColumnData] = await config.tables.save(
+      newColumn,
+      true
+    )
+
+    //Add Row to table
+    let newRow = generateNewRowForTable(<string>addColumnData._id)
+    await config.rows.add(<string>addColumnData._id, newRow)
+
+    //Search single row
+    await config.rows.searchNoPagination(<string>createdTableData._id, searchBody(<string>createdTableData.primaryDisplay))
+
+    //Add 10 more rows
```

Comment:
NAB: Worth doing this in a bulk operation

Reply:
how do bulk operations work for rows?